src/utils.h

```diff
@@ -868,6 +863,20 @@ bool WEAPON_FOCUS_USABLE_BY(struct obj_data *focus, struct char_data *ch);
 
 // ITEM_DRUG convenience defines
 
+#define GET_DRUG_TYPE(part)                        (GET_OBJ_VAL((part), 0))
+
+#define GET_DRUG_AFFECT(ch)                        (ch->player_specials->drug_affect[0])
```

Comment:
Moved them because that was the ITEM_DRUG section and I added a new one there. Thought we were grouping those per item type for easy reference but if it's only for GET_OBJ_VALs I can move them back.

Reply:
Yeah, please do, this will confuse me later otherwise. Thanks!